apis/placement/v1alpha1/disruptionbudget_types.go

```diff
@@ -44,8 +44,7 @@ type PlacementDisruptionBudgetSpec struct {
 	// If a percentage is specified, Fleet will calculate the corresponding absolute values
 	// as follows:
 	// * if the linked Placement object is of the PickFixed placement type,
-	//   the percentage is against the number of clusters specified in the placement (i.e., the
-	//   length of ClusterNames field in the placement policy);
+	//   we don't perform any calculation because eviction is not allowed for PickFixed CRP.
```

Comment:
what about pickAll?

Reply:
PickAll API comments

For MaxUnavailable: https://github.com/Azure/fleet/blob/main/apis/placement/v1alpha1/disruptionbudget_types.go#L49-L50

For MinAvailable:
https://github.com/Azure/fleet/blob/main/apis/placement/v1alpha1/disruptionbudget_types.go#L79-L80